### DISCUSSION TITLE
Is there a Rubocop command to auto-correct a specific file and then print the contents?

### DISCUSSION THREAD

**Discussion by joeldrapper** [rubocop/rubocop#11617] [Q&A]
I’d like to configure [Zed](https://zed.dev) to run Rubocop auto-corrects when saving a file. It seems to use the output as the file contents, and I can get it working with this script but I’d love to find a way that doesn't require an extra script on each project.

```
#!/usr/bin/env ruby
# frozen_string_literal: true

`rubocop -A #{ARGV[0]}`

print File.read(ARGV[0])
```

**Comment by airled** [comment]
Hi there. Seems like there is no option for this at the time without additional tools. The best solution without additional scripts would be something like this:
```
cat $(rubocop -f files -A myfile.rb)
```
